packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -85,9 +85,15 @@ export default class TableResize implements EditorPlugin {
             case PluginEventType.ContentChanged:
             case PluginEventType.Scroll:
             case PluginEventType.ZoomChanged:
+            case PluginEventType.SelectionChanged:
                 this.setTableEditor(null);
                 this.invalidateTableRects();
                 break;
+            case PluginEventType.ExtractContentWithDom:
+                e.clonedRoot.querySelectorAll('div.tableEditFeature').forEach(div => {
```

Comment:
Make this class name a constant, and only declare once, then let all other places import it. So we only have one single definition.

And better use a more specified name, such as "_roosterjsTableEditHelper"

Reply:
Which 'all other places'?

Reply:
For example, Line 27 of TableResizer.ts:

![image](https://github.com/microsoft/roosterjs/assets/23065085/1f500236-b556-4e12-a480-16654bb13044)

And Line 39 of TableSelector.ts:

![image](https://github.com/microsoft/roosterjs/assets/23065085/83871c3f-30a3-4187-8df8-c0bfdcadbe61)